### PR TITLE
fix(input layout bepo): remap visual-line-mode-map

### LINKED ISSUE #5002: t and s are not mapped properly in the bepo layout module
**What did you expect to happen?**
When using the layout module with bepo, pressing `t` and `s` should respectively call `evil-next-visual-line` and `evil-previous-visual-line` (as `j` and `k` do when not using the layout module).

**What actually happened?**
`t` and `s` respectively call `evil-next-line` and `evil-previous-line`.

**Additional details:**
I also noticed that in `text-mode` (and all "submodes" like `org-mode` and `latex-mode`), `j` is still mapped to `evil-next-visual-line` (whereas `k` is not). Perhaps it is linked (I will open a new issue concerning this only if we discover it is not linked)

**Steps to reproduce:**
1. Add `(layout +bepo)` in the `:input` section of `init.el`
2. Quit emacs and run doom sync
3. Launch emacs
4. Press `SPC h k`
5. Press `t`
6. A buffer will open indicating that the `t` key is mapped to `evil-next-line`

**System information:**
https://pastebin.com/HVVXAGBT

### PR DESCRIPTION
**Pull request by gagbo**
Fixes #5002

Draft because untested.

### CODE REVIEW

**Comment by gagbo** [comment]
Ready for review : to squash before merging

File: modules/input/layout/+bepo.el [hunk 1]
```diff
@@ -73,6 +73,23 @@ In all cases, 'h' functions go to 'c' and 'l' ones go to 'r' so the navigation k
...
+      (map! :map visual-line-mode-map
+            :m "t" 'evil-next-visual-line
+            ;; _Not_ remapping gj and gk because they aren't remapped
+            ;; consistently across all Emacs.
+            :m "s" 'evil-previous-visual-line
+            :m "È" 'evil-beginning-of-visual-line
+            :m "gÈ" 'evil-beginning-of-line
+            :m "$" 'evil-end-of-visual-line
+            :m "g$" 'evil-end-of-line
+            :m "V" 'evil-visual-screen-line)))
```
Comment (hlissner) [inline modules/input/layout/+bepo.el:92]: Please sharp-quote functions (e.g. `#'evil-next-visual-line`)

**Review by hlissner — CHANGES_REQUESTED** [review]
Save for the annotated change, code seems alright to me, so if it works for you this should be good to go.

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
Thanks again!

**Pull request merged**